src/main/scala/filter2.scala

```diff
@@ -1,19 +1,18 @@
 package era7bio.db.rna16s
 
 import era7bio.db._, csvUtils._, collectionUtils._
-
 import ohnosequences.fastarious.fasta._
 import ohnosequences.statika._
-
 import com.github.tototoshi.csv._
 import better.files._
 
-case object filter2 extends FilterDataFrom(filter1)() {
+// TODO I don't understand what this is trying to do, and how.
```

Comment:
> Now it's another way round..

I don't think so; let me reword it a bit then: If you didn't wrote the code in https://github.com/ohnosequences/db.rna16s/commit/1bccc3f18ad08fee1aed82bcccc5be0df000507b, the comments you added doesn't help much in understanding how this is dropping redundant assignments; what I'm asking you to write is a general description of the implementation, not only comments for some vals inside a method.